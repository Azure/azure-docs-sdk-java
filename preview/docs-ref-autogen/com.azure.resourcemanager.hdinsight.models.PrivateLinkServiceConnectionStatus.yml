### YamlMime:JavaType
uid: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus"
fullName: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.REJECTED"
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.REMOVED"
methods:
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.hdinsight.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-hdinsight:1.0.0-beta.5
