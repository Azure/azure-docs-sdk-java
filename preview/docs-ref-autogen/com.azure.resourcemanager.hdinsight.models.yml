### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.hdinsight.models"
fullName: "com.azure.resourcemanager.hdinsight.models"
name: "com.azure.resourcemanager.hdinsight.models"
summary: "Package containing the data models for HDInsightManagementClient. HDInsight Management Client."
classes:
- "com.azure.resourcemanager.hdinsight.models.ApplicationGetEndpoint"
- "com.azure.resourcemanager.hdinsight.models.ApplicationGetHttpsEndpoint"
- "com.azure.resourcemanager.hdinsight.models.ApplicationListResult"
- "com.azure.resourcemanager.hdinsight.models.ApplicationProperties"
- "com.azure.resourcemanager.hdinsight.models.Autoscale"
- "com.azure.resourcemanager.hdinsight.models.AutoscaleCapacity"
- "com.azure.resourcemanager.hdinsight.models.AutoscaleConfigurationUpdateParameter"
- "com.azure.resourcemanager.hdinsight.models.AutoscaleRecurrence"
- "com.azure.resourcemanager.hdinsight.models.AutoscaleSchedule"
- "com.azure.resourcemanager.hdinsight.models.AutoscaleTimeAndCapacity"
- "com.azure.resourcemanager.hdinsight.models.BillingMeters"
- "com.azure.resourcemanager.hdinsight.models.BillingResources"
- "com.azure.resourcemanager.hdinsight.models.ClientGroupInfo"
- "com.azure.resourcemanager.hdinsight.models.ClusterCreateParametersExtended"
- "com.azure.resourcemanager.hdinsight.models.ClusterCreateProperties"
- "com.azure.resourcemanager.hdinsight.models.ClusterDefinition"
- "com.azure.resourcemanager.hdinsight.models.ClusterDiskEncryptionParameters"
- "com.azure.resourcemanager.hdinsight.models.ClusterGetProperties"
- "com.azure.resourcemanager.hdinsight.models.ClusterIdentity"
- "com.azure.resourcemanager.hdinsight.models.ClusterIdentityUserAssignedIdentities"
- "com.azure.resourcemanager.hdinsight.models.ClusterListResult"
- "com.azure.resourcemanager.hdinsight.models.ClusterMonitoringRequest"
- "com.azure.resourcemanager.hdinsight.models.ClusterPatchParameters"
- "com.azure.resourcemanager.hdinsight.models.ClusterResizeParameters"
- "com.azure.resourcemanager.hdinsight.models.ComputeIsolationProperties"
- "com.azure.resourcemanager.hdinsight.models.ComputeProfile"
- "com.azure.resourcemanager.hdinsight.models.ConnectivityEndpoint"
- "com.azure.resourcemanager.hdinsight.models.DataDisksGroups"
- "com.azure.resourcemanager.hdinsight.models.DiskBillingMeters"
- "com.azure.resourcemanager.hdinsight.models.DiskEncryptionProperties"
- "com.azure.resourcemanager.hdinsight.models.EncryptionInTransitProperties"
- "com.azure.resourcemanager.hdinsight.models.Errors"
- "com.azure.resourcemanager.hdinsight.models.ExecuteScriptActionParameters"
- "com.azure.resourcemanager.hdinsight.models.FilterMode"
- "com.azure.resourcemanager.hdinsight.models.HardwareProfile"
- "com.azure.resourcemanager.hdinsight.models.JsonWebKeyEncryptionAlgorithm"
- "com.azure.resourcemanager.hdinsight.models.KafkaRestProperties"
- "com.azure.resourcemanager.hdinsight.models.LinuxOperatingSystemProfile"
- "com.azure.resourcemanager.hdinsight.models.LocalizedName"
- "com.azure.resourcemanager.hdinsight.models.NetworkProperties"
- "com.azure.resourcemanager.hdinsight.models.OperationDisplay"
- "com.azure.resourcemanager.hdinsight.models.OperationListResult"
- "com.azure.resourcemanager.hdinsight.models.OsProfile"
- "com.azure.resourcemanager.hdinsight.models.PrivateLink"
- "com.azure.resourcemanager.hdinsight.models.QuotaCapability"
- "com.azure.resourcemanager.hdinsight.models.QuotaInfo"
- "com.azure.resourcemanager.hdinsight.models.RegionalQuotaCapability"
- "com.azure.resourcemanager.hdinsight.models.RegionsCapability"
- "com.azure.resourcemanager.hdinsight.models.ResourceProviderConnection"
- "com.azure.resourcemanager.hdinsight.models.Role"
- "com.azure.resourcemanager.hdinsight.models.RoleName"
- "com.azure.resourcemanager.hdinsight.models.RuntimeScriptAction"
- "com.azure.resourcemanager.hdinsight.models.ScriptAction"
- "com.azure.resourcemanager.hdinsight.models.ScriptActionExecutionHistoryList"
- "com.azure.resourcemanager.hdinsight.models.ScriptActionExecutionSummary"
- "com.azure.resourcemanager.hdinsight.models.ScriptActionsList"
- "com.azure.resourcemanager.hdinsight.models.SecurityProfile"
- "com.azure.resourcemanager.hdinsight.models.SshProfile"
- "com.azure.resourcemanager.hdinsight.models.SshPublicKey"
- "com.azure.resourcemanager.hdinsight.models.StorageAccount"
- "com.azure.resourcemanager.hdinsight.models.StorageProfile"
- "com.azure.resourcemanager.hdinsight.models.UpdateGatewaySettingsParameters"
- "com.azure.resourcemanager.hdinsight.models.Usage"
- "com.azure.resourcemanager.hdinsight.models.VersionSpec"
- "com.azure.resourcemanager.hdinsight.models.VersionsCapability"
- "com.azure.resourcemanager.hdinsight.models.VirtualNetworkProfile"
- "com.azure.resourcemanager.hdinsight.models.VmSizeCompatibilityFilter"
- "com.azure.resourcemanager.hdinsight.models.VmSizeCompatibilityFilterV2"
- "com.azure.resourcemanager.hdinsight.models.VmSizesCapability"
enums:
- "com.azure.resourcemanager.hdinsight.models.AsyncOperationState"
- "com.azure.resourcemanager.hdinsight.models.DaysOfWeek"
- "com.azure.resourcemanager.hdinsight.models.DirectoryType"
- "com.azure.resourcemanager.hdinsight.models.HDInsightClusterProvisioningState"
- "com.azure.resourcemanager.hdinsight.models.OSType"
- "com.azure.resourcemanager.hdinsight.models.ResourceIdentityType"
- "com.azure.resourcemanager.hdinsight.models.Tier"
interfaces:
- "com.azure.resourcemanager.hdinsight.models.Application"
- "com.azure.resourcemanager.hdinsight.models.Application.Definition"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.Blank"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.WithEtag"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.hdinsight.models.Application.DefinitionStages.WithTags"
- "com.azure.resourcemanager.hdinsight.models.Applications"
- "com.azure.resourcemanager.hdinsight.models.BillingResponseListResult"
- "com.azure.resourcemanager.hdinsight.models.CapabilitiesResult"
- "com.azure.resourcemanager.hdinsight.models.Cluster"
- "com.azure.resourcemanager.hdinsight.models.Cluster.Definition"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.Blank"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.hdinsight.models.Cluster.DefinitionStages.WithTags"
- "com.azure.resourcemanager.hdinsight.models.Cluster.Update"
- "com.azure.resourcemanager.hdinsight.models.Cluster.UpdateStages"
- "com.azure.resourcemanager.hdinsight.models.Cluster.UpdateStages.WithTags"
- "com.azure.resourcemanager.hdinsight.models.ClusterConfigurations"
- "com.azure.resourcemanager.hdinsight.models.ClusterMonitoringResponse"
- "com.azure.resourcemanager.hdinsight.models.Clusters"
- "com.azure.resourcemanager.hdinsight.models.Configurations"
- "com.azure.resourcemanager.hdinsight.models.Extension"
- "com.azure.resourcemanager.hdinsight.models.Extensions"
- "com.azure.resourcemanager.hdinsight.models.GatewaySettings"
- "com.azure.resourcemanager.hdinsight.models.HostInfo"
- "com.azure.resourcemanager.hdinsight.models.Locations"
- "com.azure.resourcemanager.hdinsight.models.Operation"
- "com.azure.resourcemanager.hdinsight.models.Operations"
- "com.azure.resourcemanager.hdinsight.models.RuntimeScriptActionDetail"
- "com.azure.resourcemanager.hdinsight.models.ScriptActions"
- "com.azure.resourcemanager.hdinsight.models.ScriptExecutionHistories"
- "com.azure.resourcemanager.hdinsight.models.UsagesListResult"
- "com.azure.resourcemanager.hdinsight.models.VirtualMachines"
metadata: {}
package: "com.azure.resourcemanager.hdinsight.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-hdinsight:1.0.0-beta.1
