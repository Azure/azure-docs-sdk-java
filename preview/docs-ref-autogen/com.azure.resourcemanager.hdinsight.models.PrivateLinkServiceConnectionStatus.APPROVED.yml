### YamlMime:JavaMember
uid: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.hdinsight.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.hdinsight.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-hdinsight:1.0.0-beta.5
